apis/leds/src/tests.rs

```diff
@@ -0,0 +1,85 @@
+use libtock_unittest::fake;
+
+type Leds = super::Leds<fake::Syscalls>;
+
+#[test]
+fn no_driver() {
+    let _kernel = fake::Kernel::new();
+    assert!(Leds::count().is_none());
```

Comment:
If you use `assert_eq!(Leds::count(), None);`, then an assertion failure will print out the returned count, which may be useful debugging information.